deepchem/utils/rdkit_utils.py

```diff
@@ -703,3 +707,660 @@ def ordered_pair(a, b):
             rings_dict[atom_pair].append((ring_sz, is_aromatic))
 
   return rings_dict
+
+
+class DescriptorsNormalizationParameters:
+  """
+  A class for holding a dictionary of 200 RDKit descriptors and the corresponding distribution functions (based on `scipy.stats` module nomenclature)
+  and associated parameters required for creating normalized descriptor functions.
+
+  (The distribution functions and parameters data was collected from the source code of 'descriptastorus' library)
+  """
```

Comment:
Reference for values

Reply:
I have added the link in the latest commit